Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml

```diff
@@ -0,0 +1,862 @@
+  display: Custom Filter
+  hidden: false
+  name: custom_filter
+  required: false
+  type: 0
+description: Microsoft Cloud App Security is a multimode Cloud Access Security Broker
+  (CASB). It provides rich visibility, control over data travel, and sophisticated
+  analytics to identify and combat cyber threats across all your cloud services.
+display: Microsoft Cloud App Security
+name: MicrosoftCloudAppSecurity
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: |-
+        Multiple alerts matching the specified filters.
+        The alert_id should appear similar to "55af7415f8a0a7a29eef2e1f".
+        Mandatory, unless you use a custom filter.
+      isArray: true
+      name: alert_ids
+      required: false
+      secret: false
+    - default: false
+      description: 'Filter that you create yourself. If you use this custom filter,
+        other filters will not work. The filter template can be found here: https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters.'
+      isArray: false
+      name: custom_filter
+      required: false
+      secret: false
+    - default: false
+      description: Comment about why the alerts are dismissed.
+      isArray: false
+      name: comment
+      required: false
+      secret: false
+    deprecated: false
+    description: Command to dismiss multiple alerts matching the specified filters.
+    execution: false
+    name: microsoft-cas-alert-dismiss-bulk
+  - arguments:
+    - default: false
+      description: Skips the specified number of records.
+      isArray: false
+      name: skip
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '50'
+      description: The maximum number of records returned to the user.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The severity of the alert.
+      isArray: false
+      name: severity
+      predefined:
+      - Low
+      - Medium
+      - High
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Filter by alert resolution status.
+      isArray: false
+      name: resolution_status
+      predefined:
+      - Open
+      - Dismissed
+      - Resolved
+      required: false
+      secret: false
+    - default: false
+      description: 'Filter that you create yourself. If you use this custom filter,
+        other filters will not work. The filter template can be found at: https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters.'
+      isArray: false
+      name: custom_filter
+      required: false
+      secret: false
+    - default: false
+      description: The alert ID
+      isArray: false
+      name: alert_id
+      required: false
+      secret: false
+    deprecated: false
+    description: Prints a list of the alerts.
```

Comment:
Not sure I'd use the word `prints`, it's a programming word and not suitable for a user-facing description.